iphone/Classes/TiUIiOSSplitWindow.m

```diff
@@ -108,6 +108,7 @@ - (void)layoutSubviewsForOrientation:(UIInterfaceOrientation)orientation
 {
   CGSize refSize = self.bounds.size;
   BOOL isPortrait = UIInterfaceOrientationIsPortrait(orientation);
```

Comment:
@adamtarmstrong  Thanks for PR. It is fixing the first issue, which you have raised.

Rather introducing a new variable 'isRunningInFullScreen', can we modify above line as -
`  BOOL isPortrait = UIApplication.sharedApplication.keyWindow.frame.size.height > UIApplication.sharedApplication.keyWindow.frame.size.width;` and no other change. This will ensure that app is in portrait mode on basis of height and width of key window. Or do you think there is any issue with this? Thanks!

Reply:
I'm not exactly sure what you're asking.  You know AppC/Titanium better than I do.  If you need to tweak anything please go for it.  I just proposed the small changes I made and tested on my end.